test/lib/mayaUsd/render/pxrUsdMayaGL/testProxyShapeDrawColors.py

```diff
@@ -21,30 +21,36 @@
 import sys
 import unittest
 
+
 class testProxyShapeDrawColors(unittest.TestCase):
+
     @classmethod
     def setUpClass(cls):
         # The test USD data is authored Z-up, so make sure Maya is configured
         # that way too.
         cmds.upAxis(axis='z')
 
-        cmds.loadPlugin('pxrUsd')
+        cmds.loadPlugin('mayaUsdPlugin')
```

Comment:
We still explicitly load `mayaUsdPlugin` here because the Maya scene doesn't have a `requires` line for it, and because we use `mayaUSDExport()` and `mayaUSDImport()` below.